CHANGELOG.next.asciidoc

```diff
@@ -52,6 +52,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 
 - Setting period for counter cache for Prometheus remote_write at least to 60sec {pull}38553[38553]
 - Add support of Graphite series 1.1.0+ tagging extension for statsd module. {pull}39619[39619]
+- Remove fallback to the node limit for the `kubernetes.pod.cpu.usage.limit.pct` and `kubernetes.pod.memory.usage.limit.pct` metrics calculation
```

Comment:
I would say that This should me moved under the `Breaking Changes` section

Reply:
It is already under `Breaking changes` > `Metricbeat` https://github.com/elastic/beats/blob/873aebc393d753d1ccd9cea9cca141067b368a3a/CHANGELOG.next.asciidoc#breaking-changes
am I missing smth?

Reply:
I see this, so maybe it is a rendering issue of the page. I though is only under Metricbeat
![Screenshot 2024-06-27 at 4 38 35 PM](https://github.com/elastic/beats/assets/1685415/70c980c0-9986-4924-a4ec-9d1fe18c1302)